fiat-google-groups/src/test/groovy/com/netflix/spinnaker/fiat/roles/google/GoogleDirectoryUserRolesProviderSpec.groovy

```diff
@@ -0,0 +1,72 @@
+package com.netflix.spinnaker.fiat.roles.google
+
+import com.netflix.spinnaker.fiat.permissions.ExternalUser
+import com.google.api.services.admin.directory.model.Group;
+import com.google.api.services.admin.directory.model.Groups;
+import spock.lang.Specification
+
+class GoogleDirectoryUserRolesProviderSpec extends Specification {
+    GoogleDirectoryUserRolesProvider.Config config = new GoogleDirectoryUserRolesProvider.Config()
+
+    def "should read google groups"() {
+        setup:
+        config.domain = "test.com"
+        def group = new Group()
```

Comment:
Can get rid of the setters: ` def group = new Group(name: "test", email: "test@test.com")`

Reply:
i am not great with groovy :P